### PR TITLE
Refactor

### LINKED ISSUE #34: Refactor existing structures to new standard

### PR DESCRIPTION
**Pull request by cjdenio**
Closes #34

### CODE REVIEW

File: frameworks/deno/README.md [hunk 1]
```diff
@@ -1,35 +1,5 @@
...
-export * from "https://deno.land/x/oak/mod.ts";
-```
-Then, import your dependencies elsewhere:
-```ts
-import { serve, Application } from "./deps.ts";
-```
\ No newline at end of file
+| Structure | Description |
+|-----------|-------------|
+| [Basic Package Structure](basic/) | A starting point for a Deno package, written in TypeScript. |
```
Comment (Matt-Gleich) [inline frameworks/deno/README.md:5]: Below this add the following:
`<!--END OF TOC, DO NOT REMOVE-->`

**Comment by Cal-Hagner** [comment]
Do we want to replace the icons with inline images in here while we're at it?

**Comment by cjdenio** [comment]
That's something we should save for a future PR. It's actually kind of difficult :confused:

File: frameworks/deno/README.md (hunk 1 above)
Comment (Matt-Gleich) [reply frameworks/deno/README.md:5]: Please do that to everyone. Thanks!

**Comment by Matt-Gleich** [comment]
Another thing, refactor tree to look like the `tree` command. We can do this now or just use the CLI. What do you guys think?

**Comment by cjdenio** [comment]
I'll just do the `tree` refactor now.

**Comment by sampoder** [comment]
It was already merged awesome!!! I'm blind good job everyone

**Pull request merged**